src/content/docs/guides/Adonis.md

```diff
@@ -86,3 +86,57 @@ Dokumentacja Adonisa proponuje następujące wtyczki
 ### Webstorm
 
 Po zainstalowaniu Adonisa Webstorm powinien to wykryć i zaproponować nam instalację wtyczki [Adonis](https://plugins.jetbrains.com/plugin/22932-adonis).
+
+## Migracje
+
+### Ogólne
+
+Migracje służą do zmieniania struktury bazy danych. **Podczas ich pisania należy zwrócić uwagę na o żeby funcja `down()` odwracała całkowicie działanie funkcji `up()`. Niezastosowanie się do tej uwagi może nam przynieść dużo problemów przy ewentualnym niepowodzeniu migracji**
+
+### Kolejność
+
+Migracje wykonywane są w kolejności alfabetycznej (jeżeli nie zmienialiście nazw jest to kolejność tworzenia). Trzeba o tym pamiętać tworząc migracje, które są zależne od siebie. Najpierw tworzymy migrację od której będą zależeć inne, dopiero potem te zależne. W przypadku pomyłki najłatwiej zmienić nazwę.
+
+### Enumy w PostgreSQL
+
+Jeżeli korzystamy z postgresa (a tak powinno być), to deklarując enumy należy skorzystać z natywnego enuma oferowanego przez bazę danych.
+
+```jsx
+this.schema.createTable('users', (table) => {
+  table.enu('account_status', ['PENDING', 'ACTIVE', 'SUSPENDED'], {
```

Comment:
literówka chyba

Reply:
jeżeli chodzi o `enu` zamiast `enum` to adonis obydwie nazwy wspiera
kod prosto z dokumentacji